src/components/TreeView.js

```diff
@@ -34,6 +40,18 @@ export default class TreeView extends React.Component {
   }
 }
 
+function checkSignature(nodes, fetchedSigners) {
+  const signatureBuffer = Buffer.from(nodes[1].value.value, 'base64');
+  const sig = fetchedSigners.filter(x => x.sig.equals(signatureBuffer))[0];
+  
+  let isValid = SIGNATURE.INVALID;
+  if (sig) {
+    isValid = sig.isValid;
+  }
```

Comment:
`isValid` defaults to `SIGNATURE.INVALID` so in case of errors (ex. connection error) the signature will be displayed as invalid (you can go to Chrome Dev tools, switch to Offline mode and load a `TransactionEnvelope` to reproduce it).

What do you think something like this:
* When checking the signatures (loading signers etc) we add "Checking signatures..." to `TransactionEnvelope.signatures` node:
![screen shot 2018-08-14 at 15 43 50](https://user-images.githubusercontent.com/464938/44095366-e4e69802-9fd8-11e8-9b43-68b67058d73e.png)
* If there was an error we display it at the `TransactionEnvelope.signatures` level:
![screen shot 2018-08-14 at 15 45 28](https://user-images.githubusercontent.com/464938/44095437-195db1c4-9fd9-11e8-90a1-9a34b580d26b.png)
* If checking was successful we display the statuses (like we do now):
![screen shot 2018-08-14 at 15 45 52](https://user-images.githubusercontent.com/464938/44095486-3ea61d4a-9fd9-11e8-9e73-c92376a5c464.png)

Reply:
Looks good! So basically a sort of visual of the state of fetched signatures:
* loading
* failed
* success (same as now)